packages/ui/src/ui/components/ActiveText/ActiveText.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import cn from 'bem-cn-lite';
+
+import './ActiveText.scss';
+
+const block = cn('yt-active-text');
+
+export type ActiveTextProps = {
+    className?: string;
+    /**
+     * 'info' is default value
+     */
+    color?: 'info' | 'secondary' | 'primary';
+    onClick?: (e: React.MouseEvent) => void;
+    children: React.ReactNode;
+    title?: string;
+};
+
+export function ActiveText({className, color = 'info', onClick, children}: ActiveTextProps) {
```

Comment:
The name does little to reflect what the component does. What is active text? You may get the feeling that this is some kind of animation. I suggest LinkButton. Or TextButton